src/ort_genai.h

```diff
@@ -279,12 +279,26 @@ struct OgaTensor : OgaAbstract {
 };
 
 struct OgaImages : OgaAbstract {
-  static std::unique_ptr<OgaImages> Load(const char* image_path) {
+  static std::unique_ptr<OgaImages> Load(const std::vector<const char*>& image_paths) {
     OgaImages* p;
-    OgaCheckResult(OgaLoadImage(image_path, &p));
+    OgaStringArray* strs;
+    OgaCheckResult(OgaCreateStringArrayFromStrings(image_paths.data(), image_paths.size(), &strs));
+    OgaCheckResult(OgaLoadImages(strs, &p));
+    OgaDestroyStringArray(strs);
     return std::unique_ptr<OgaImages>(p);
   }
 
+#if __cplusplus >= 202002L
```

Comment:
Why is this `if` check needed?

Reply:
std::span is not available in c++ 17. This #if checks if the compiler language is c++20 or above.